src/components/ContentWidgets/ToDoList/ToDoListEditor.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { WidgetEditorProps } from "../../ContentMapping/ContentMapping";
+import { ToDoList } from "./ToDoList";
+
+/**
+ * To do list editor allows making changes to a list of checked or unchecked items
+ *
+ * Last Modified
+ * Jennifer Tao
+ * July 6, 2019
+ */
+export const ToDoListEditor: React.FC<WidgetEditorProps> = ({
+  editedContent,
+  setEditedContentOnChange
+}) => {
+  let toDoList_content = editedContent.toDoList_content || [{}];
```

Comment:
are you sure you want the thing to be an array with a single object inside it if the content doesn't exist?